api/api.go

```diff
@@ -18,8 +18,6 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-// GQLErrorsCollection is a slice of errors returned by the GraphQL server.
-// Each error is made up of a GQLResponseError type.
```

Comment:
Just making sure, as it's not connected to the main PR's topic: was this removal intentional?

Reply:
Correct, this was removed in 43a2846 so we could see an actual notification in slack